### ISSUE TITLE
[combiner] VDS combiner needs to support call_fields argument

### ISSUE THREAD

**Issue by tpoterba** [hail-is/hail#13382]
### What happened?

Encountered this making the repo to support the SVCR paper.

There's no way to pass through multiple call fields through to `hl.import_gvcf_interval`, so I'm stuck with a PGT str. Should be a simple fix.

### Version

0.2.120

### Relevant log output

```shell
Entry fields:
    'END': int32
    'DP': int32
    'GQ': int32
    'LA': array<int32>
    'LGT': call
    'LAD': array<int32>
    'LPGT': str
```

**Comment by chrisvittal** [comment]
This case in particular was fixed in #13364 but Yeah, the handling needs to be better.

**Comment by jmarshall** [comment]
Is this the same issue as #13337?

**Comment by chrisvittal** [comment]
Not quite. This is a feature request asking to add a `call_fields` argument to the combiner.

**Comment by jmarshall** [comment]
Thanks. So did I waste my time writing up #13337, or did that report help contribute to that part getting fixed in #13364?

**Comment by danking** [comment]
This was resolved by @chrisvittal in https://github.com/hail-is/hail/pull/13385 and released in 0.2.121. Use
```python3
combiner = hl.vds.new_combiner(..., call_fields=YOUR_CUSTOM_CALL_FIELDS_ARRAY)
combiner.run()
```
The combiner also, by default, recognizes `LPGT`.
